packages/dropdowns/src/elements/Menu/Items/MediaFigure.spec.tsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { render } from 'garden-test-utils';
+import { Dropdown, Trigger, Menu, MediaItem, MediaBody, MediaFigure } from '../../..';
+
+describe('MediaFigure', () => {
+  it('passes ref to underlying DOM element', () => {
+    const ref = React.createRef<HTMLImageElement>();
+
+    const { getByTestId } = render(
+      <Dropdown>
+        <Trigger>
+          <button data-test-id="trigger">Test</button>
+        </Trigger>
+        <Menu>
+          <MediaItem value="image">
+            <MediaFigure>
+              <img ref={ref} data-test-id="media-figure" src="test.png" alt="test" />
```

Comment:
I noticed that the `ref` is applied to the native `img` tag. Should this be on the `MediaFigure` component instead?

Reply:
The `MediaFigure` component using `React.cloneElement` to apply styling directly onto an element. This test is ensuring that all props are spread to the correct element (itself).